README.md

```diff
@@ -8,74 +8,10 @@ These form part of the software development standards policy for Defra and apply
 
 All developers working on behalf of Defra need to be aware of their obligation to follow these standards.
```

Comment:
This page could do with making some reference to the fact that the content has moved slightly

Reply:
It does but if you missed it, then others will.  I've moved the section about how best to view the guide further up the page and also made the references to these standards in this section a link to it also.

Reply:
I was thinking of maybe something more explicitly geared towards people that had perhaps saved the link (see how optimistic I am!) or were otherwise directed to this page, e.g. from our policy doc in Content Cloud.

Something just to say "we've moved" - so they know why it looks different?

Reply:
I've added a note to the top of the readme in the repo to explain where best to view them and one to the top of the hosted pages to explain they're sourced from this repo.  That work?

Reply:
Yeah let's go with that and see if anyone notices!

I reckon it should be fine but we can always revise it later if we get any queries 👍